aws/app/ecs_task/form_viewer.json

```
@@ -89,6 +89,10 @@
       {
         "name": "AUDIT_LOG_QUEUE_URL",
         "value": "${audit_log_queue_url}"
+      },
+      {
+        "name": "ZITADEL_PROVIDER",
+        "valueFrom": "${zitadel_provider}"
```

This just needs to be like so:
```suggestion
        "value": "${zitadel_provider}"
```